engine/src/main/java/org/terasology/particles/ParticlePool.java

```diff
@@ -225,6 +240,74 @@ public void storeTemporaryDataAt(final int index, final int rawMask) {
         }
     }
 
+    /**
+     * Initializes this particle pool for rendering.
+     * The method should be called right after object creation.
+     */
+    public void initRendering() {
```

Comment:
The only reason for me to not do this automatically in the constructor is that the `glMethods` are breaking the unit tests, because they run without an OpenGL context.